changelogs/unreleased/gh-9098-out-of-bound-write-in-error_payload_destroy.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* Fixed a possible out-of-bound write on unpacking a MsgPack error extension
+  (gh-9098).
```

Comment:
I think it's worth mentioning that only debug builds are affected.

Reply:
Done.